src/Parser/Shaman/Restoration/Modules/Abilities.js

```diff
@@ -40,8 +40,8 @@ class Abilities extends CoreAbilities {
       },
       {
         spell: SPELLS.HEALING_STREAM_TOTEM_CAST,
-        category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
-        charges: 2,
+        category: Abilities.SPELL_CATEGORIES.DEFENSIVE,
```

Comment:
I just pushed a commit so you don't have to change the category of this spell to include it as a defensive and can instead just set the `isDefensive` prop to `true` - if you want.